docs/api/mirror-gateway.md

```diff
@@ -0,0 +1,161 @@
+# Vald Mirror Gateway APIs
+
+## Overview
+
+Mirror Service is responsible for providing a register and an advertise interface for the Vald Mirror Gateway.
```

Comment:
```suggestion
Mirror Service is responsible for providing `Rregister` and `Advertise` interface for the Vald Mirror Gateway.
```

Reply:
BTW there is a typo on ykadowak suggestion.
Rregister -> register